Pagination: describe API (#6707)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxPagination/dxPagination.md

---
id: dxPagination
module: ui/pagination
export: default
inherits: Widget
---
---
##### shortDescription

> Pagination is a UI component that allows users to navigate through pages and change their size at runtime. Pagination consists of the page navigator and several optional elements: the page size selector, navigation buttons, and page information.

Reviewer: Since we're just introducing a component and its parts, we can refer to elements with indefinite article. "Pagination UI includes a page navigator". We don't really know yet what that navigator is like. We're only introducing it. 

```suggestion
Pagination is a UI component that allows users to navigate through pages and change page size at runtime. Pagination UI includes a page navigator and several optional elements: a page size selector, navigation buttons, and page information.
```
